functions/index.js

```diff
@@ -4,6 +4,37 @@ const cors = require('cors')({origin: true});
 const admin = require('firebase-admin');
 admin.initializeApp();
 
+exports.saveAudioSuggestions = functions.https.onRequest(async (req, res) => {
+  const uuidv1 = require('uuid/v1');
+  console.log('Saving audio to GCS.');
+  var bucket = admin.storage().bucket('barnard-project-audio');
+  const options = {
+    metadata: {
+      contentType: 'audio/mpeg',
+    }
+  };
+  var uuid = uuidv1();
+  var file = bucket.file(`suggestions/${uuid}.mp3`);
+  await file.save(req.body, options)
+  .then(stuff => {
+      console.log('Audio saved successfully.');
+      return file.getSignedUrl({
+          action: 'read',
+          expires: '03-09-2500'
+        })
+  })
+  .then(urls => {
+      var url = urls[0];
+      console.log(`Audio url = ${url}`)
+      res.status(200).send(JSON.stringify(url));
+      return url
+  })
+  .catch(err => {
+      console.log(`Unable to upload audio ${err}`)
+  });
+});
+
+
 exports.addSuggestions = functions.https.onRequest(async (req, res) => {
   console.log('Add a suggestion');
```

Comment:
Are these logs intentional or just for debugging?

Reply:
For debugging